sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/application-offering-change-request.controller.service.ts

```diff
@@ -0,0 +1,53 @@
+import { Injectable, NotFoundException } from "@nestjs/common";
+import { ApplicationOfferingChangeRequestService } from "../../services";
+import { ApplicationOfferingChangesAPIOutDTO } from "./models/application-offering-change-request.dto";
+import { getUserFullName } from "../../utilities";
+
+@Injectable()
+export class ApplicationOfferingChangeRequestControllerService {
+  constructor(
+    private readonly applicationOfferingChangeRequestService: ApplicationOfferingChangeRequestService,
+  ) {}
+
+  /**
+   * Get the Application Offering Change Request by its id.
+   * @param id the Application Offering Change Request id.
+   * @param options method options:
+   * - `studentId`: student id for student authorization.
+   * - `locationId`: location id for institution authorization.
+   * @returns application offering change request.
+   */
+  async getById(
```

Comment:
IMO, maybe we can give a proper name like `getApplicationOfferingChangeRequest`

Reply:
The reason I used `getById` was because the part `ApplicationOFferingChangeRequest` is implied from the file name. @dheepak-aot @andrepestana-aot @guru-aot suggestions?

Reply:
That's ok to be just `id` as it is from  `application-offering-change-request.controller.service.ts` and it is commented.

Reply:
`getApplicationOfferingChangeRequest`  will be more preferable considering any method in controller service is treated like method written in controller. 

But it is not a blocker IMO.